wicket-core/src/main/java/org/apache/wicket/Page.java

```diff
@@ -827,9 +827,10 @@ public void component(final Component component, final IVisit<Void> visit)
 			getSession().getPageManager().touchPage(this);
 		}
 
+		//TODO WICKET-6890 What about TAG_ATTRIBUTE?
 		if (getApplication().getDebugSettings().isOutputMarkupContainerClassName())
 		{
-			String className = Classes.name(getClass());
+			final String className = Classes.name(getClass());
 			getResponse().write("<!-- Page Class ");
 			getResponse().write(className);
 			getResponse().write(" END -->\n");
```

Comment:
What should we do here? Ignore the strategy and always write an HTML comment, or only write it when the `HTML_COMMENT` strategy is enabled?